client/web/src/components/app.tsx

```diff
@@ -10,12 +10,13 @@ import { ReactComponent as TailscaleIcon } from "src/icons/tailscale-icon.svg"
 import ProfilePic from "src/ui/profile-pic"
 import { Link, Route, Router, Switch, useLocation } from "wouter"
 import DeviceDetailsView from "./views/device-details-view"
+import { UpdatingView } from "./views/updating-view"
 
 export default function App() {
   const { data: auth, loading: loadingAuth, newSession } = useAuth()
 
   return (
-    <main className="min-w-sm max-w-lg mx-auto py-14 px-5">
+    <main className="min-w-sm max-w-lg mx-auto py-14 px-5 h-screen">
```

Comment:
the UpdatingView needs this to take the entire screen height - otherwise it just takes up as much space as the content at the top of the screen, rather than being centered in the scren

Reply:
I would use top margin to move it down the screen a little; i'm scared this might get in the way of page scrolls in the future

Reply:
would `min-h-screen` be alright in that case? the thing with margins is that I don't think tailwind has a way to do margins that are aware of screen size, so it'll look off-center vertically on any screen size that's too different from mine

Reply:
I think it's alright to not be totally center vertically - as long as it's centered horizontally.